### PR TITLE
volpy maskrcnn compatible tensorflow 2.4.1

### PR DESCRIPTION
**Pull request by caichangjia**
# Description

Mask R-CNN in VolPy previously only worked on tensorflow 1 and needed users to install keras. Now it worked on tensorflow2.4.1.

Fixes # (887, 963)

# Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: caiman/source_extraction/volpy/mrcnn/model.py [hunk 1]
```diff
@@ -2378,7 +2368,7 @@ def train(self, train_dataset, val_dataset, learning_rate, epochs, layers,
             validation_steps=self.config.VALIDATION_STEPS,
             max_queue_size=100,
             workers=workers,
-            use_multiprocessing=True,
+            use_multiprocessing=workers > 1,
```
Comment (pgunn) [inline caiman/source_extraction/volpy/mrcnn/model.py:2371]: I see above you mention that Keras fails on Windows when using multiprocessing; is this still true of the keras layer in tensorflow? If not, the comment above (line 2363) should be removed. If not, is this code nonusable on windows with this change?

**Review by pgunn — COMMENTED** [review]
Generally looks solid, and it's great news that we won't be stuck on TF1.x.

I left a few comments.

File: caiman/source_extraction/volpy/mrcnn/model.py [hunk 2]
```diff
@@ -8,38 +8,34 @@
...
+import tensorflow.keras.layers as KL
+import tensorflow.keras.layers as KE
+import tensorflow.keras.utils as KU
+from tensorflow.python.eager import context
+import tensorflow.keras.models as KM
 
 from ..mrcnn import utils
 
-# Requires TensorFlow 1.3+ and Keras 2.0.8+.
+# Requires TensorFlow 2.0+
```
[the remark this answers is not in the document]
Comment (caichangjia) [reply caiman/source_extraction/volpy/mrcnn/model.py:28]: You mean update tensorflow to 2.0+ in the environment.yml file? Yes, we can do this. 
We are using tensorflow 2.4.1 in the current version of CaImAn anyway.

File: caiman/source_extraction/volpy/mrcnn/model.py (hunk 1 above)
Comment (caichangjia) [reply caiman/source_extraction/volpy/mrcnn/model.py:2371]: I actually only tested the Mask R-CNN in VolPy on the linux system.
